.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.0
```

Comment:
The `v4` tag should be moving to the latest release. That is, if `v4.1.0` is the latest, then `v4` should be strictly equal to `v4.1.0`.

Reply:
Just extra information: `v4` doesn't yet exist because when we released it Dependabot started updating all the repos immediately, so for now the latest version should be manually used